documentation/sdk/docs/how_tos/simple_transaction/simple_transaction.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Simple Transaction
+description: 'How to send a value transaction'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- faucet
+- testnet
+- test funds
+- nodejs
+- python
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import RustCode from "../../libraries/how_tos/simple_transaction/simple_transaction/rust/_code_sample.mdx";
+import NodejsCode from "../../libraries/how_tos/simple_transaction/simple_transaction/nodejs/_code_sample.mdx";
+import PythonCode from "../../libraries/how_tos/simple_transaction/simple_transaction/python/_code_sample.mdx";
+import Output from "../../libraries/how_tos/simple_transaction/simple_transaction/_expected_output.mdx";
+import TipFaucet from "../../_admonitions/_tip_faucet.md";
+
+Once you have created your account<!-- TODO [created your account]() --> and
+[requested funds from the faucet](./request_funds.mdx) you are ready to send your first
+transaction.
+
+<TipFaucet/>
+
+## Code Example
+
+The following example will:
+
+1. Create a wallet.
+2. Get Alice's account which was created in the first guide<!-- TODO "created in the first guide" link-->.
+3. Send 1 SMR to an address using the `Account.send_amount(addresses_with_amount, options)` function. Make sure to update the code sample to send to the address of your choice.
+
+
+<Tabs groupId="language">
+    <TabItem value="rust" label="Rust">
+        <RustCode/>
+    </TabItem>
+    <TabItem value="nodejs" label="Nodejs">
+        <NodejsCode/>
+    </TabItem>
+    <TabItem value="python" label="Python">
+        <PythonCode/>
+    </TabItem>
+</Tabs>
```

Comment:
@lucas-tortora what do you think about such a change? Get rid of all the _sample_code.mdx files, because I think they're useless and only more annoying since we have to touch multiple files
```suggestion
import RustCode from '!!raw-loader!../../../../../../../sdk/examples/how_tos/simple_transaction/request_funds.rs';
import NodejsCode from '!!raw-loader!../../../../../../../bindings/nodejs/examples/how_tos/simple_transaction/request_funds.ts';
import PythonCode from '!!raw-loader!../../../../../../../bindings/python/examples/how_tos/simple_transaction/request_funds.py';
import Output from "../../libraries/how_tos/simple_transaction/simple_transaction/_expected_output.mdx";
import TipFaucet from "../../_admonitions/_tip_faucet.md";

Once you have created your account<!-- TODO [created your account]() --> and
[requested funds from the faucet](./request_funds.mdx) you are ready to send your first
transaction.

<TipFaucet/>

## Code Example

The following example will:

1. Create a wallet.
2. Get Alice's account which was created in the first guide<!-- TODO "created in the first guide" link-->.
3. Send 1 SMR to an address using the `Account.send_amount(addresses_with_amount, options)` function. Make sure to update the code sample to send to the address of your choice.


<Tabs groupId="language">
    <TabItem value="rust" label="Rust">
        <CodeBlock className="language-rust">
            {RustCode}
        </CodeBlock>
    </TabItem>
    <TabItem value="nodejs" label="Nodejs">
        <CodeBlock className="language-typescript">
            {NodejsCode}
        </CodeBlock>
    </TabItem>
    <TabItem value="python" label="Python">
        <CodeBlock className="language-typescript">
            {PythonCode}
        </CodeBlock>
    </TabItem>
</Tabs>
```